examples/md-flexible/src/configuration/MDFlexConfig.cpp

```diff
@@ -477,6 +479,10 @@ void MDFlexConfig::calcSimulationBox() {
   resizeToObjectLimits(sphereObjects);
   resizeToObjectLimits(cubeClosestPackedObjects);
 
+  if (boxMin.value != preBoxMin or boxMax.value != preBoxMax) {
+    std::cout << "WARNING: Simulation box increased due to particles being too close to the boundaries." << std::endl;
+  }
```

Comment:
If you want, you could add the information from which value it changed. You could also omit this because the user could deduce this by comparing the input with the log output. But since you are targeting users who are not aware of this, they might not think about doing this.

Reply:
The configuration log output is pretty much directly beneath the warning, which is why I would omit it for now.